Description: Don't list deprecated source to translate.
Author: Sebastien Bacher <seb128@ubuntu.com>

Index: shotwell-0.22.0/po/POTFILES.in
===================================================================
--- shotwell-0.22.0.orig/po/POTFILES.in
+++ shotwell-0.22.0/po/POTFILES.in
@@ -226,5 +226,4 @@ src/util/string.vala
 src/util/system.vala
 src/util/ui.vala
 [type: gettext/glade]ui/set_background_dialog.glade
-[type: gettext/glade]ui/set_background_slideshow_dialog.glade
 [type: gettext/glade]ui/shotwell.glade
